src/utilities/calculateDailyEarnings.spec.ts

```diff
@@ -0,0 +1,16 @@
+import BigNumber from 'bignumber.js';
+import { Asset } from 'types';
+import { assetData as assets } from '__mocks__/models/asset';
+import calculateDailyEarningsCents from './calculateDailyEarnings';
+import { calculateYearlyEarningsForAssets } from './calculateYearlyEarnings';
+
+describe('utilities/calculateDailyEarningsCentss', () => {
+  test('calculates yearly Earnings for array of assets', () => {
+    const earnings = calculateYearlyEarningsForAssets({
+      assets: assets as Asset[],
+      borrowBalanceCents: new BigNumber('90.0103018').multipliedBy(100),
+      isXvsEnabled: true,
+    });
+    expect(calculateDailyEarningsCents(earnings.yearlyEarningsCents || new BigNumber(0))).toBe(5);
```

Comment:
Because we only want to test the util here, I'd just pass a value directly:
```suggestion
    expect(new BigNumber(1825).toFixed()).toBe('5');
```